src/bepasty/views/delete.py

```diff
@@ -4,23 +4,22 @@
 from flask.views import MethodView
 from werkzeug.exceptions import NotFound, Forbidden
 
-from ..constants import *  # noqa
-
-from ..utils.permissions import *
+from .. import constants
+from ..utils import permissions
 from ..utils.http import redirect_next_referrer
 
 
 class DeleteView(MethodView):
     def post(self, name):
-        if not may(DELETE):
+        if not permissions.may(permissions.DELETE):
```

Comment:
same/similar here. old was easier to read.

Reply:
Importing everything (*) from `permissions` shows how bad it is. You didn't even realize that in `src/bepasty/apis/rest.py` the `request` and `current_app` weren't even imported directly form flask, but from `permissions` - that's why I had to add their imports.